library/cwm/src/lib/cwm/abstract_widget.rb

```diff
@@ -188,6 +188,13 @@ def focus
       Yast::UI.SetFocus(Id(widget_id))
     end
 
+    # Determines whether the widget is currently displayed in the UI
+    #
+    # @return [Boolean]
+    def widget_present?
```

Comment:
I am not sure if widget is not duplicte. Maybe simple `present?` will be enough?

Reply:
Yes, I agree. But I fear that `present?` is already used (see https://github.com/yast/yast-network/blob/master/src/lib/y2network/widgets/wireless_scan_button.rb#L61). However, I could simply use "displayed?" or "drawn?"... or change the `WirelessScanButton`.

Reply:
I like that `displayed?` That sounds good for me.

Reply:
and when we are at it, you can extend documenation of value/value= by mentioning that it makes sense to call it only when displayed? is true